docs/index.js

```diff
@@ -225,7 +229,11 @@ const App = () => {
             </TopBarItem>
           </MenuBar>
         </TopBar>
-        <SideBar show={state.show} onDismiss={onTriggerClick}>
+        <SideBar
+          show={state.show}
+          onDismiss={onTriggerClick}
+          className="SideBar-with-footer"
```

Comment:
Since this a modifier of `SideBar`, I suggest this:
```suggestion
          className="SideBar--with-footer"
```